src/test/java/com/egov/core-services/pretests/updateUserPasswordPretest.feature

```diff
@@ -0,0 +1,144 @@
+Feature: eGovUser-passwordUpdate
+
+  Background:
+     * def jsUtils = read('classpath:jsUtils.js')
+     * def userProfileData = read('../constants/user.yaml')
+     # Calling access token
+     * def authUsername = userProfileData.password.userName
+     * def authPassword = userProfileData.password.password
+     * def tenantId = superUserTenantId
+     * print tenantId
+     * def authUserType = superUserAuthUserType
+     * call read('../pretests/authenticationToken.feature')
+     * def updatedUserPassword = read('../requestPayload/user/updatePassword/updatePassword.json')
+     * set updatedUserPassword.existingPassword = userProfileData.validPasswordDetails.existingPassword
+     * set updatedUserPassword.newPassword = userProfileData.validPasswordDetails.newPassword
+     * set updatedUserPassword.tenantId = userProfileData.validPasswordDetails.tenantId
+     * set updatedUserPassword.username = userProfileData.password.userName
+
+  @validPassword
+  Scenario: Update existing password with valid password
+   * configure headers = read('classpath:websCommonHeaders.js') 
+    Given url updatePassword
+     * print updatePassword
+    And request updatedUserPassword
+     * print updatedUserPassword
+    When method post
+    And  def updatedPasswordResponseBody = response
+
+  @inValidNewPassword
+  Scenario: Update existing password with invalid new password
+   * configure headers = read('classpath:websCommonHeaders.js') 
+   * set updatedUserPassword.newPassword = userProfileData.inValidPasswordDetails.newInvalidPassword
+    Given url updatePassword
+     * print updatePassword
+    And request updatedUserPassword
+     * print updatedUserPassword
+    When method post
+    Then status 400
+    And  def updatedPasswordResponseBody = response
+     * print updatedPasswordResponseBody
+
+  @inValidExistingPassword
+  Scenario: Update existing password with invalid existing password
+   * configure headers = read('classpath:websCommonHeaders.js') 
```

Comment:
@krishnendu-eGov - Why are we using headers in all pre tests? We can keep it in background.